File: String/codeforces_734A.py

# Problem link ===>> https://codeforces.com/problemset/problem/734/A

def anton_and_danik(n, s):
  """Returns the winner of the game, or "Friendship" if it is a tie."""
  anton = 0
  danik = 0
  for char in s:
    if char == 'A':
      anton += 1
    elif char == 'D':
      danik += 1

  if anton > danik:
    return "Anton"
  elif anton < danik:
    return "Danik"
  else:
    return "Friendship"

n = int(input())
s = input()

winner = anton_and_danik(n, s)
print(winner)

File: String/codeforces_734A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/734/A
// submission link ===>> https://codeforces.com/contest/734/submission/224940292

#include <iostream>
using namespace std;

int main()
{
    int n;
    string s;
    cin >> n >> s;

    int anton = 0, danik = 0;
    for (int i = 0; i < n; i++)
    {
        if (s[i] == 'A')
        {
            anton++;
        }
        else
        {
            danik++;
        }
    }

    if (anton > danik)
    {
        cout << "Anton" << endl;
    }
    else if (anton < danik)
    {
        cout << "Danik" << endl;
    }
    else
    {
        cout << "Friendship" << endl;
    }

    return 0;
}
